src/components/ai/AIConfig.vue

```diff
@@ -136,10 +99,10 @@ async function testConnection() {
     }
 
     const payload = {
-      model: config.model || (currentService().models[0] || ``),
+      model: config.model,
       messages: [{ role: `user`, content: `ping` }],
-      temperature: 0,
-      max_tokens: 1,
+      temperature: config.temperature,
+      max_tokens: config.maxToken,
```

Comment:
```suggestion
      temperature: 0,
      max_tokens: 1,
```

testConnection 的时候，max_tokens 这些我是有意设置成很小的值的，可以减少 token 消耗。这里只需要确保接口是通的